### ISSUE TITLE
[BUG] Braille Symbols behave poorly

### ISSUE THREAD

**Issue by felixjulianheitmann** [aristocratos/btop#139]
**Describe the bug**

The screenshot below shows how the graphs seem to use the wrong symbols. This is consistent across all graphs and across my available terminals. tty mode behaves fine.

**To Reproduce**

I use alacritty with "Fira Mono for Powerline" as font [`from their github page`](https://github.com/powerline/fonts). The issue however also occurs within VS Codes integrated terminal and with other fonts such as Google's RobotoMono, and the default monospace.
BTop version is at 1.0.21 and I compiled from source.

**Expected behavior**

As previously on my other machine, I would expect the graphs to not have arbitrary outlier dots. The only correct braille symbol seems to be the full 6-dot block.

**Screenshots**

![image](https://user-images.githubusercontent.com/75343504/140236006-53cdfa58-4266-4f07-9cb5-ff9520fcffbe.png)

**Info (please complete the following information):**
 - btop++ version: 1.0.21
 - Binary: self compiled
 - (If compiled) Compiler and version: g++ 11.1.0
 - Architecture: x86_64
 - Platform: Linux - arch
 - (Linux) Kernel: 5.14.15-arch1-1
 - Terminal used: alacritty 0.9.0
 - Font used: Fira Mono for Powerline / RobotoMono / monospace

**Additional context**

Logs are empty.

```
2021/11/04 (01:24:19) | ===> btop++ v.1.0.21
2021/11/04 (01:24:19) | DEBUG: Starting in DEBUG mode!
2021/11/04 (01:24:19) | INFO: Logger set to DEBUG
2021/11/04 (01:24:19) | DEBUG: Using locale de_DE.UTF-8
2021/11/04 (01:24:19) | INFO: Running on /dev/pts/1
2021/11/04 (01:24:19) | DEBUG: Loading theme file: /usr/local/share/btop/themes/gruvbox_dark.theme
2021/11/04 (01:24:29) | INFO: Quitting! Runtime: 00:00:10
```

**Comment by aristocratos** [comment]
@aristocratos 
Try running the precompiled static binary from release, if that works fine this would be an issue in your build toolchain.
Also try the font referenced in the README.md [Terminess Powerline](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Terminus/terminus-ttf-4.40.1).
Otherwise this would be an issue in your systems font configuration.

All the braille symbols are `const` so will never change after compilation, the only possibility for this to happen is if your system isn't properly decoding the UTF-8 byte sequences.

If this is a global issue on your system the code excerpt below will likely also look wrong, first row should be 0-4.

https://github.com/aristocratos/btop/blob/45169223ebb6ed792b92e8c74bdf5b281712d1e0/src/btop_draw.cpp#L44-L89

**Comment by felixjulianheitmann** [comment]
You're right. My PC does show the source code differently from my phone for example. And it was the font I used. Obiously, I didn't read the README thoroughly enough.
I am now using FiraCode Nerd Font Mono and it works flawlessly.

Thank you for the help and the awesome tool!
